src/components/Tools/UsefulLinks.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="links">
+    <div v-for="{ title, link } in list" :key="title" class="links-group">
+      <a class="links-link" :href="link">
+        {{ title }}
+        <img class="links-link-icon" src="@/assets/images/link-gray.svg" alt="external link" />
+      </a>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  computed: {
+    list(): { link: string; title: string }[] {
+      return [
+        {
+          title: 'Student Center',
+          link: 'https://studentcenter.cornell.edu/',
+        },
+        {
+          title: 'Course Catalog',
+          link: 'https://courses.cornell.edu/',
+        },
+        {
+          title: 'Cornell DTI',
+          link: 'https://www.cornelldti.org/',
+        },
+      ];
```

Comment:
@michaelfarkouh @zachary0kent any idea if we will add links in the future? Or if the links will be user-specific (eg. we link CS majors to the CS homepage and CS Student Checklist)? If so, after this release it might be good to think about refactoring this data so that it's not hardcoded in a vue file.

Also, would Scheduler be a good addition to this list?

Reply:
Scheduler is definitely good to add. I wasn't really sure what links we wanted so I just picked some. We could definitely make this more dynamic in the future (if someone wants to go link fishing 🎣)